src/components/v5/shared/ProgressBar/ProgressBar.tsx

```diff
@@ -12,6 +12,10 @@ const ProgressBar: FC<ProgressBarProps> = ({
   barClassName,
   className,
 }) => {
+  if (threshold == null || threshold === 0) {
```

Comment:
Why non-strict equality? If you want to cover null, undefined and 0 you could consider `!threshold`, unless of course you want to allow empty strings